public/index.php

```diff
@@ -67,3 +77,11 @@
  */
 
 $app->run();
+
+// Save Config Cache
+// $factoriesCache->save('config');
```

Comment:
can this be auto detected by env, eg: enabled when `CI_ENVIRONMENT` on "production" ?

Reply:
If a developer changes a Config value for a particular visitor while the app is running, that value will be cached.

If s/he always changes the value for each visitor, this is not a problem, but if does not, the cached value for one visitor will be used for another.

Also, simply changing the Config file will not change the cached values.

After all, there are many conditions for using this cache, and it is not for everyone.
So I only want it to be used by people who understand what is going on.

Reply:
imo cache should be enabled on production by default. If someone want to debug on production, they should enable "development" env temporary to clear cache and fix the bug, when fix, enabling "production" again should refill the cache to be reused.

Reply:
It is never safe to toggle development in production since our Debug Toolbar and error handling can reveal sensitive data. We have `spark` available for cache troubleshooting, and if we need an additional solution it should be added as a CLI command.

Reply:
Ok, fine then 👍